packages/ember-metal/lib/run_loop.js

```diff
@@ -683,3 +683,55 @@ run._addQueue = function(name, after) {
     run.queues.splice(run.queues.indexOf(after) + 1, 0, name);
   }
 };
+
+run._eventCallbacks = {};
+
+function bindEventName(eventName) {
+  return function trigger(arg1, arg2) {
+    let callbacks = run._eventCallbacks[eventName];
+    if (callbacks) {
+      for (let i = 0; i < callbacks.length; ++i) {
+        callbacks[i](arg1, arg2);
+      }
+    }
+  };
+}
+
+for (let eventName in backburner._eventCallbacks) {
```

Comment:
since this is internal, and we are only using begin, it could have just been begin.